### ISSUE TITLE
Current package cannot be added on Windows

### ISSUE THREAD

**Issue by 1jerry** [ongoworks/meteor-google-spreadsheets#7]
When attempting to add your package from Windows I get:

```
 => Errors while adding packages:

While checking for ongoworks:google-spreadsheets@0.4.0:
error: No compatible binary build found for this package. Contact the package author and ask
them to publish it for your platform.
```

Version 0.1.0 is the most recent version I can add, but this version doesn't have the 0.4.0 updates to node-google.spreadsheets necessary to work with Googles newer API.

**Comment by balasivagnanam** [comment]
+1

**Comment by aaronjudd** [comment]
I'd like to publish but haven't had time to debug the following publishing error:

```
 C:\Windows\system32>node-gyp rebuild
'node-gyp' is not recognized as an internal or external command,
operable program or batch file.
```

Seems similar to one that has been reported by others trying to publish on windows: https://github.com/meteor/meteor/issues/4087

I recently resolved a similar issue locally, but since this error is on the Meteor build box, I'm not sure what the workaround is.

**Comment by ktruckenmiller** [comment]
Can't upgrade on mac either. Said new version is available but no dice.

```
$ meteor add ongoworks:google-spreadsheets@0.4.2
 => Errors while adding packages:             

While checking for ongoworks:google-spreadsheets@0.4.2:
error: No compatible binary build found for this package. Contact the package author and ask them to publish it for your
platform.
```

**Comment by aaronjudd** [comment]
just republished `0.4.2_1` for osx, linux 32/64.  Windows is still failing.

**Comment by gouthamve** [comment]
Any chance you could give this a shot now @aaronjudd ?

gouthamve/reaction-cod#8 depends on this.

**Comment by aaronjudd** [comment]
Looks like Meteor finally fixed the Windows build system.  Published `ongoworks:google-spreadsheets@0.4.2_3` for all platforms successfully.
